core/contracts/financial-templates/implementation/demo/DepositBox.sol

```diff
@@ -0,0 +1,346 @@
+pragma solidity ^0.6.0;
+pragma experimental ABIEncoderV2;
+
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+
+import "../FeePayer.sol";
+import "../../../common/implementation/FixedPoint.sol";
+
+import "../../../oracle/interfaces/IdentifierWhitelistInterface.sol";
+import "../../../oracle/interfaces/OracleInterface.sol";
+import "../../../oracle/interfaces/AdministrateeInterface.sol";
+import "../../../oracle/implementation/ContractCreator.sol";
+
+
+/**
+ * @title Token Deposit Box
```

Comment:
small nit: try get the line length to be close to 120 chars so they are all conistant with our other contracts large comment boxes. This'll save a line or 2 at the end as well.

Reply:
what would you change here for example?

Reply:
was a comment about the large block of text below this line. Try get each line to end just before the 120 line mark so they are all consistent.

Reply:
got it. WIll do. I thought the linter would handle this though?